tests/test_filepath.py

```diff
@@ -127,7 +124,7 @@ def test_filepath_class(table=Filepath(), store="repo"):
     assert_true(table.external[store])
 
     # delete from external table
-    table.external[store].delete()
+    table.external[store].delete(delete_external_files=True)
 
 
 def test_filepath_class_again():
```

Comment:
- This test failed on WIN10 at `test_filepath_class_again`. Error details: `datajoint.errors.DataJointError: A different version of 'one\two\three\attachment.dat' has already been placed`
- This test failed on WIN10 at `test_filepath_class_s3`. For example, the result of `managed_file` was `dj/repo/one\two\three\attachment.dat`.
- This test failed on WIN10 at `test_filepath_class_s3_again`. For example, the result of `managed_file` was `dj/repo/one\two\three\attachment.dat`.